manifests/config.pp

```diff
@@ -18,13 +18,13 @@
     $debug_header = ''
     $debug_footer = ''
   }
-  $config_dir = $::riemann::config_dir
-  $config_include_dir = $::riemann::config_include_dir
+  $config_dir = $riemann::config_dir
```

Comment:
is the form `$::deprecated` now ?

Reply:
Yes, since Puppet 4 I think.  The old parser was confused without it but it's not needed anymore.